twisted/names/test/test_rootresolve.py

```diff
@@ -209,20 +209,20 @@ def test_lookupChecksClass(self):
         badClass.CLASS = HS
         servers = {
             ('1.1.2.3', 53): {
-                ('foo.example.com', A): {
-                    'answers': [('foo.example.com', badClass)],
-                    'authority': [('foo.example.com', Record_NS('ns1.example.com'))],
-                    'additional': [('ns1.example.com', Record_A('10.0.0.2'))],
+                (b'foo.example.com', A): {
+                    'answers': [(b'foo.example.com', badClass)],
+                    'authority': [(b'foo.example.com', Record_NS(b'ns1.example.com'))],
+                    'additional': [(b'ns1.example.com', Record_A('10.0.0.2'))],
```

Comment:
Why is the argument to `Record_A` a `str` but the argument to `Record_NS` is a bytes?